docs/reference-guide/modules/events/pages/infrastructure.adoc

```diff
@@ -53,9 +116,11 @@ Configure your application:
 
 [source,java]
 ----
-// Returns a Configurer instance with default components configured.
+import org.axonframework.messaging.core.configuration.MessagingConfigurer;
+
+// Returns a MessagingConfigurer instance with default components configured.
 // `AxonServerEventStore` is configured as Event Store by default.
-Configurer configurer = DefaultConfigurer.defaultConfiguration();
+MessagingConfigurer configurer = MessagingConfigurer.create();
```

Comment:
To get an event store, the `EventSourcingConfigurer` is required, not the `MessagingConfigurer`.